guides/metrics.md

```diff
@@ -0,0 +1,108 @@
+# Metrics
+
+MongoosePush 2.1 provides metrics in the Prometheus format on the `/metrics` endpoint.
+This is a breaking change compared to previous releases.
+Existing dashboards will need to be updated.
+
+It is important to know that metrics are created inside MongoosePush only when a certain event happens.
+This may mean that a freshly started MongoosePush node will not have all the possible metrics available yet.
+
+## Available metrics
+
+#### Histograms
```

Comment:
I looked at how the page is rendered on the github pages and noticed that the headings are not aligned. For instance, this one jumps two levels from the previous one.
```suggestion
### Histograms
```